src/applications/personalization/dashboard-2/tests/unit/appointments.unit.spec.jsx

```diff
@@ -0,0 +1,91 @@
+// Ensure sorting is done correctly (startsAt wise)
+// Ensure we display the correct location for CC and VA
+// Ensure we show correct additional Info for video
+// Ensure we display correct timezone
+
+import React from 'react';
+import { mount } from 'enzyme';
+import { expect } from 'chai';
+
+import Appointments from '~/applications/personalization/dashboard-2/components/health-care/Appointments';
+
+import {
+  upcomingVAAppointment,
+  upcomingCCAppointment,
+  upcomingVideoAppointment,
+  farFutureAppointments,
+} from '~/applications/personalization/dashboard-2/utils/mocks/appointments';
+
+describe('Appointments', () => {
+  describe('when we have an upcoming VA appointment', () => {
+    const props = {
+      authenticatedWithSSOe: true,
+      appointments: upcomingVAAppointment,
+    };
+    const wrapper = mount(<Appointments {...props} />);
+
+    it('should render all necessary elements', () => {
```

Comment:
This testing coverage is very basic and we're going to need to add more specific unit tests for used helpers (i.e. timezone logic and video appointment type)